harpy/_misc.py

```
@@ -84,15 +85,34 @@ def fetch_report(workdir, target):
-        else:
-            print_error("report configuration missing", f"The required quarto configuration file [blue bold]_quarto.yml[/blue bold] was not found within the Harpy installation.")
-            print_solution("There may be an issue with your Harpy installation, which would require reinstalling Harpy. Alternatively, there may be in a issue with your conda/mamba environment or configuration.")
-            sys.exit(1)
-
+    # pull yaml config file from github, use local if fails
+    try:
+        yaml = "https://github.com/pdimens/harpy/raw/refs/heads/main/harpy/reports/_quarto.yml"
+        destination = f"{workdir}/report/_quarto.yml"
+        with urllib.request.urlopen(yaml) as response, open(destination, 'w') as yaml_out:
+            yaml_out.write(response.read().decode("utf-8"))
+    except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)